test/integration/roles/setup_selinux/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+- name: Install packages for selinux
+  package:
+    name: libselinux-python
+    state: present
```

Comment:
That's likely. But I do not know what is the plan for testing on python3, since we do have the same problem for postgresql and mysql and anything that need a python3 library.